docs/src/content/docs/id/guides/authoring-content.md

```diff
@@ -0,0 +1,209 @@
+
+```md
+---
+title: Halaman Konten Saya
+description: Cara menggunakan _anchor links_ bawaan Starlight
+---
+
+## Pengantar
+
+Saya dapat menautkan ke [kesimpulan saya](#kesimpulan) di bagian bawah halaman yang sama.
+
+## Kesimpulan
+
+`https://my-site.com/halaman1/#pengantar` langsung menuju ke Pengantar saya.
+```
+
+Judul Level 2 (`<h2>`) dan Level 3 (`<h3>`) akan secara otomatis muncul di daftar isi halaman.
+
+## Asides
+
+_Asides_ (juga sering disebut sebagai _“admonitions”_ atau “_callouts”_) berguna untuk menampilkan informasi sekunder bersama konten utama halaman.
+
+Starlight menyediakan sintaks Markdown kustom untuk merender _asides_. Blok _asides_ ditandai dengan sepasang tiga titik dua `:::` untuk melingkupi konten Anda, dan dapat berjenis `note`, `tip`, `caution`, atau `danger`.
+
+Anda dapat menyusun berbagai jenis konten Markdown lainnya di dalam sebuah _asides_, tetapi _asides_ lebih cocok untuk potongan konten yang pendek dan padat.
+
+### Catatan Sampingan
+
+:::note
+Starlight adalah toolkit website dokumentasi yang dibangun dengan [Astro](https://astro.build/). Anda dapat memulai dengan perintah ini:
+
+```sh
+npm create astro@latest -- --template starlight
+```
+
+:::
+
+````md
+:::note
+Starlight adalah toolkit website dokumentasi yang dibangun dengan [Astro](https://astro.build/). Anda dapat memulai dengan perintah ini:
+
+```sh
+npm create astro@latest -- --template starlight
+```
+
+:::
+````
+
+### Judul Asides Kustom
+
+Anda dapat menentukan judul kustom untuk _asides_ dengan menambahkan tanda kurung siku setelah jenis _asides-nya_, misalnya `:::tip[Apakah Anda tahu?]`.
+
+:::tip[Apakah Anda tahu?]
+Astro membantu Anda membangun website lebih cepat dengan “Arsitektur Pulau”.
+:::
+
+```md
+:::tip[Apakah Anda tahu?]
+Astro membantu Anda membangun website lebih cepat dengan “Arsitektur Pulau”.
+:::
+```
+
+### Jenis Asides Lainnya
+
+_Asides_ berjenis _caution_ dan _danger_ berguna untuk menarik perhatian pengguna pada detail-detail yang mungkin membuat mereka bingung.
+Jika Anda sering menggunakan ini, mungkin juga pertanda bahwa hal yang Anda dokumentasikan sepertinya bisa di-desain ulang.
+
+:::caution
+Jika Anda tidak yakin ingin membuat situs dokumen yang menakjubkan, pikirkan dua kali sebelum menggunakan [Starlight](../../).
+:::
+
+:::danger
+Pengguna Anda mungkin lebih produktif dan menemukan produk Anda lebih mudah digunakan berkat fitur-fitur Starlight yang membantu.
+
+- Navigasi yang jelas
+- Tema warna yang dapat dikonfigurasi oleh pengguna
+- [i18n support](/guides/i18n)
+
+:::
+
+```md
+:::caution
+Jika Anda tidak yakin ingin membuat situs dokumen yang menakjubkan, pikirkan dua kali sebelum menggunakan [Starlight](../../).
+:::
+
+:::danger
+Pengguna Anda mungkin lebih produktif dan menemukan produk Anda lebih mudah digunakan berkat fitur-fitur Starlight yang membantu.
+
+- Navigasi yang jelas
+- Tema warna yang dapat dikonfigurasi oleh pengguna
+- [i18n support](/guides/i18n)
```

Comment:
One small update needed to these links so they point to the `/id/` version:

```suggestion
- [i18n support](/id/guides/i18n)

:::

```md
:::caution
Jika Anda tidak yakin ingin membuat situs dokumen yang menakjubkan, pikirkan dua kali sebelum menggunakan [Starlight](../../).
:::

:::danger
Pengguna Anda mungkin lebih produktif dan menemukan produk Anda lebih mudah digunakan berkat fitur-fitur Starlight yang membantu.

- Navigasi yang jelas
- Tema warna yang dapat dikonfigurasi oleh pengguna
- [i18n support](/id/guides/i18n)
```